db/migrate/20231002150909_change_association_between_location_and_adopter_profile.rb

```diff
@@ -0,0 +1,35 @@
+class ChangeAssociationBetweenLocationAndAdopterProfile < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def up
+    safety_assured do
```

Comment:
I wrapped everything inside `safety_assured` because strong_migrations were complaining about splitting this code into multiple deployments. I think this was completely unnecessary for a system that does not have a production environment yet.